lib/ukrand/include/sys/random.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef _SYS_RANDOM_H
+#define _SYS_RANDOM_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sys/types.h>
+
+#define GRND_NONBLOCK     0x01
+#define GRND_RANDOM       0x02
+
+ssize_t getrandom(void *buf, size_t buflen, unsigned int flags);
```

Comment:
Why do we need this definition? Isn't `getrandom` a system call, or is it defined as per `libc`? As a kernel we should try and not directly use system calls. I understand that this is simply moving files but we should consider removing this IMO.